assets/js/custom.js

```diff
@@ -1 +1,29 @@
 // Put your custom JS code here
+
+// self-hosting view.
+
+const storedSelfhost = localStorage.getItem('selfhost');
+
+const getSelfhost = () => {
+  if (storedSelfhost) {
+    return storedSelfhost;
+  }
+
+  return 'no';
+};
+
+const setSelfhost = function (selfhost) {
```

Comment:
This can be `(selfhost) => { ... }` instead of a function literal.

Unrelated, could we also update `localStorage` in this function?

Reply:
Thanks for the tip! sure, that sounds reasonable.